tasties/urls.py

```diff
@@ -24,5 +24,7 @@
     path('', views.recipes, name="recipes"),
     path('login/', views.login_user, name="login"),
     path('register/', views.register, name="register"),
-    path('logout/', views.logout_user, name="logout")
+    path('logout/', views.logout_user, name="logout"),
+    # path('view_recipe/<int:recipe_id>/', views.single_recipe, name="single_recipe"),
```

Comment:
So this comment going replaced in another PR?

Reply:
Yes, when we'll have the ability to enter a recipe from the recipe's menu.
I'll remove it for now.